doc/SimpleWeatherService.md

```diff
@@ -43,6 +45,8 @@ The byte array must contain the following data:
    - 6 = Thunderstorm
    - 7 = Snow
    - 8 = Mist, smog
+  - [49, 50] : Sunrise (16 bits, number of minutes elapsed since midnight)
+  - [51, 52] : Sunset (16 bits, number of minutes elapsed since midnight)
```

Comment:
What about if there is no sunrise or sunset e.g midnight sun at northern latitudes?

Reply:
Great question. This data would be sent from a companion and/or weather app, so it all depends how the apps do this calculation.
Likely it'd be empty (which would cast to 0) or have a negative number. Both cases would lead to the watch showing as daytime.

Reply:
Do we want to support the case where it's always night (e.g polar winter). If it gets complex and messy it might not be worth implementing

Reply:
I don't think it's worth implementing. It would require some knowledge of _why_ the sunrise/sunset data is not reported (or reported "differently"), rather than assuming the meaning of an empty/0, which I think is beyond the scope of this app.

Also in the case of a polar winter, worst case here is that weather is reported with a sun icon rather than a moon icon, which is the current behavior anyway (until this is merged at least), so it'd be a no-op in that scenario. 😛

Reply:
OK cool in that case the only suggestion I have is that the protocol specification should be clear about the expected value when there is no sunrise or sunset, and also when it's unknown.

I would suggest u16 max and then counting backwards (e.g 65535 for unknown, 65534 for no sunset/sunrise event that day) as that allows more flexibility if we want to care more about this later on.
Since there are less than 2^15 minutes in a day, it may also be worth making the field signed in this case as then it's just -1 / -2 which are easier to work with

With this protocol the implementation here could just bail on any negative and assume it's day to avoid the complex edge cases